cybershell/config.py

```
@@ -0,0 +1,153 @@
+    return get_config().exploitation
+
+def get_llm_config() -> LLMConfig:
+    """Get current LLM configuration"""
+    return get_config().llm
+
+def get_learning_config() -> LearningConfig:
+    """Get current learning configuration"""
+    return get_config().learning
+
+# Default configuration factory
+def create_default_config(target: str = None) -> UnifiedConfig:
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)